src/global-table.ts

```diff
@@ -0,0 +1,34 @@
+import * as core from 'aws-cdk-lib';
```

Comment:
Import the specific APIs in alphabetical order you are going to use from 'aws-cdk-lib', not the whole thing. 

```suggestion
import { IResource, Stack, RemovalPolicy, Resource } from 'aws-cdk-lib';
```